manage.py

```diff
@@ -2,6 +2,8 @@
 import os
 import sys
 
+import threading; threading.stack_size(4*80*1024)
```

Comment:
A comment on why this is need would be nice

Reply:
I don't know _why_ it's needed 🙈.

Random stack overflow posts or github issues resolved it by manually setting the stack size. We should do some research on what the implications of this is.